src/main/java/com/kkukielka/springrecipeproject/domain/Ingredient.java

```diff
@@ -15,6 +15,9 @@
     @ManyToOne
     private Recipe recipe;
 
+    @OneToOne(fetch = FetchType.EAGER)
+    private UnitOfMeasure unitOfMeasure;
```

Comment:
why use EAGER fetch type if it is default for this relationship?

Reply:
I wanted for other developers to know that I intentionally set this fetch type
(I am talking to myself, cool)